src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import {  View, Text } from 'react-native'
+import styles from '../SearchBar/SearchBar.style';
+
+const SearchBar = () => {
+ const styleSheet = styles();
+    return (
+        <View style={styleSheet.searchBar}>
+            <Text>Funciona</Text>
+        </View>
+    )
+}
+
+export default SearchBar
```

Comment:
This component should be into the Field folder; The componente name could be "FieldSearchBar"

Reply:
Done